### PR TITLE
(#204) Added a map/geolocation section

### PR DESCRIPTION
**Pull request by AppleC4T**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes https://github.com/fossasia/gci18.fossasia.org/issues/204
- Preview Link: https://applec4t.github.io/gci18.fossasia.org/
Screenshot:
image

image

## Description / Changes
- Added a map section using the open-source leaflet API. Added custom markers for students and mentors. The script reads the jekyll files for coordinates and places markers that show the name of the marked person on click. If any suggestions/feedback arise let me now, but note that the jekyll loop**has** to be in the footer in order for the script to compile correctly.
- Updated my student profile

### CODE REVIEW

**Comment by AakashMallik** [comment]
screenshot-applec4t github io-2018 11 04-15-19-41

I somehow managed to do this to the section you added, if you could prevent users from doing this, it would be a much more robust UI component. By the way, good initiative. :+1:

**Comment by AppleC4T** [comment]
@AakashMallik done. You now cannot zoom out too much. Let me just squash the new commits.

**Comment by AppleC4T** [comment]
@AakashMallik and all done. Should be all fine now, thanks for the feedback 👍

**Comment by AakashMallik** [comment]
@AppleC4T That was quick... Good job.

File: _includes/header.html [hunk 1]
```diff
@@ -72,12 +78,13 @@
         <div class="module-group right">
           <div class="module left">
             <ul class="menu">
-              <li><a href="#welcome">Welcome</a></li>
-              <li><a href="#mentors">Mentors</a></li>
-              <li><a href="#students">Students</a></li>
-              <li><a href="#contributors">Contributors</a></li>
+              <li><a href="index.html#welcome">Welcome</a></li>
```
Comment (fragm3) [inline _includes/header.html]: No need to change these. We need scrolling behaviour. Undo for other nav links also.

**Comment by AppleC4T** [comment]
@fragm3 done

**Pull request merged**